### ISSUE TITLE
Reduced Potential Definition in pymbar's FES Implementation

### ISSUE THREAD

**Issue by joaco-rgb** [choderalab/pymbar#534]
Hello to all the community, I am writing to you because I have been using the FES implementation of pymbar to calculate the PMF from an umbrella sampling, but I have encountered some issues with the definition of the reduced potential.

The documentation explains that the reduced potential, or u_kn, used to initialize the FES object, is an array of shape K,N where K is the number of simulations or states and N is the number of configurations of each state. However, in the umbrella sampling example code on GitHub, the u_kln matrix, which is an array of shape K,K,N, is used to initialize the FES object. I am wondering why this matrix is used instead.

Additionally, to generate the PMF, the u_kn matrix is used, but the documentation mentions that the reduced potential of one state can be used. Could you help me understand these discrepancies?

**Comment by ftclark3** [comment]
Hello,

This has been confusing to me, too. It seems that `u_kln` is an older format and they are working to update the examples to the `u_kn` formatting (#542). The matrix `u_kln` is defined [here](https://github.com/choderalab/pymbar/blob/1eda979f87b69d6f3d83bf4c1b7a249819c59084/examples/parallel-tempering-2dfes/parallel-tempering-2dfes.py#L226), although I would read "temperature" as "thermodynamic state," encompassing changes in the temperature and/or potential. 

It seems that the burden is now on the user to concatenate the sets of configurations sampled in each state and ensure that they are ordered appropriately in `u_kn`, as stated [here](https://pymbar.readthedocs.io/en/pymbar-3-lts/mbar.html): 

> We assume that the states are ordered such that the first N_k are from the first state, the 2nd N_k the second state, and so forth.

**Comment by mrshirts** [comment]
Yeah, both formats are accepted at this point, but everything is done internally as u_kn.
